### PR TITLE
PL | Update 12.0

### PR DESCRIPTION
**Pull request by MultiverseUncle**
# **SCP:SL Translation PR Template**

## **Language Name**
Polish

## **State Type of change (New language, changed existing Etc.)**
Added and updated files for 12.0. 
Oh, also redid my previous (already closed) merge request.

# **Checklist:**
- [X] I have tested the translation changes ingame
- [X] I have made sure to not commit changes to multiple translations at the same time, or non-translation repo files
- [X] I have ran `EncodingNormalizer.exe` on my changed files before commiting 
   -  *( Exception to this is when edits has __only__ been made in-browser. If this is the case, also* ***check this box*** *)*
- [X] I have made sure that I'm **not** submitting a duplicate translation 
- [X] I have made sure my addition follows guidelines described in the [CONTRIBUTING.md](https://github.com/northwood-studios/SCPSL-Translations/blob/master/.github/CONTRIBUTING.md) file
- [X] I have made sure to label my translation folder by [Microsoft Language Tag Standards](https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-lcid/a9eac961-e77d-41a6-90a5-ce1a8b0cdb9c)
   - Only in cases of **new** translations (Can be tested [Here](https://rextester.com/WDXPS97501)). Also check this box if the change is **not** a new translation
> For more information regarding game translations, please visit our [README](https://github.com/northwood-studios/SCPSL-Translations/blob/master/README.md) page.

### CODE REVIEW

File: Translations/pl/SCP079_HUD.txt [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+STREFA WYŻSZEGO NADZORU
+STREFA WEJŚCIOWA
+POWIERZCHNIA
+ZGAŚ ŚWIATŁA ([cost])
+GASZENIE ŚWIATEŁ NIEDOSTĘPNE W TYM POMIESZCZENIU
+OSIĄGNIĘTO LIMIT ZGASZONYCH ŚWIATEŁ
+ODNAWIANIE GASZENIE ŚWIATEŁ
+WYŚLIJ WINDĘ ([cost])
+ZABLOKUJ POMIESZCZENIE ([cost])
+ODNAWIANIE BLOKADY POMIESZCZENIA
```
Comment (zabszk) [inline Translations/pl/SCP079_HUD.txt:32]: Same as line 16.

File: Translations/pl/SCP079_HUD.txt [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+BRAK DOSTĘPNYCH DRZWI DO ZABLOKOWANIA POMIESZCZENIA
+ZAZNACZ LOKACJĘ ([cost])
+OSIĄGNIĘTO LIMIT ZAZNACZEŃ
+ZMIANA KAMERY NIE POWIODŁA SIĘ
+BRAK SYGNAŁU
+ŁĄCZENIE (ZA: [time]s)
+AKTYWUJ BRAMĘ TESLI ([cost])
+ODNAWIANIE AKTYWACJI BRAMY TESLI
```
Comment (zabszk) [inline Translations/pl/SCP079_HUD.txt:40]: Same thing as previously.

File: Translations/pl/SCP079_HUD.txt [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+$Poziom dostępu [tier]$ - ODBLOKOWANY
+$ENERGIA$ - limit zwiększony do $[limit]$.
+$ZGAŚ ŚWIATŁA$ jest dostępne - naciśnij $[key]$, aby zgasić światła w pomieszczeniu.
+$ZGAŚ ŚWIATŁA$ - limit zwiększony do $[number]$ pomieszczeń.
+$ZABLOKUJ POMIESZCZENIE$ jest dostępne - naciśnij $[key]$, aby zablokować wszystkie drzwi w pomieszczeniu.
```
Comment (zabszk) [inline Translations/pl/SCP079_HUD.txt:45]: These sound a little weird and not grammatically correct.

Maybe we should use forms like "GASZENIE ŚWIATEŁ jest dostępne"? Furthermore the translations are not consistent, two of them follow the format "feature - change" and some of them "feature change" (without the dash).

File: Translations/pl/SCP939_HUD.txt [hunk 4]
```diff
@@ -0,0 +1,12 @@
+Naciśnij [key_attack] lub [key_jump] aby wyskoczyć.
```
Comment (zabszk) [inline Translations/pl/SCP939_HUD.txt:1]:
```suggestion
Naciśnij [key_attack] lub [key_jump], aby wyskoczyć.
```

File: Translations/pl/SCP939_HUD.txt [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+Odsłuchaj głos (tylko ty to usłyszysz)
+PODGLĄD
+Usuń głos (przytrzymaj)
+Transmituj głos (każdy to usłyszy)
+Zostałeś zabity przez SCP-939 i ukradł ci twój głos!
```
Comment (zabszk) [inline Translations/pl/SCP939_HUD.txt:6]: The second part of the sentence doesn't have any subject. Sentence "You have been killed by SCP-939 and stole your voice" is not correct. Furthermore "ci" in the second part is redundant.

We can either change "i" na ", który" or the entire sentence (eg. "SCP-939 cię zabił i ukradł twój głos").

File: Translations/pl/SCP939_HUD.txt [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+Jeżeli nie chcesz aby twój głos był przez niego używany, możesz wyłączyć tę mechanikę.
```
Comment (zabszk) [inline Translations/pl/SCP939_HUD.txt:7]:
```suggestion
Jeżeli nie chcesz, aby twój głos był przez niego używany, możesz wyłączyć tą funkcję.
```

File: Translations/pl/SCP939_HUD.txt [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+Zapamiętaj mój wybór
+Usuń
+OK
+Więcej informacji
+SCP-939 używa mechaniki pozwalającej na nagrywanie kilku ostatnich sekund twojej rozmowy głosowej. Ona może być odtwarzana przez SCP-939, aby zwabić do siebie pozostałych graczy.\nNagrania są automatycznie usuwane jeżeli opuścisz serwer, SCP-939 zginie lub gdy runda dobiegnie końca. Wyłączenie tej mechaniki spowoduje usunięcie wszystkich nagrań twojego głosu.
```
Comment (zabszk) [inline Translations/pl/SCP939_HUD.txt:12]:
```suggestion
SCP-939 używa mechaniki pozwalającej na nagrywanie kilku ostatnich sekund twojej rozmowy głosowej. Może być on odtwarzany przez SCP-939, aby zwabić do siebie pozostałych graczy.\nNagrania są automatycznie usuwane jeżeli opuścisz serwer, SCP-939 zginie lub gdy runda dobiegnie końca. Wyłączenie tej funkcji spowoduje usunięcie wszystkich nagrań twojego głosu.
```

File: Translations/pl/Voicechat.txt [hunk 8]
```diff
@@ -1,3 +1,19 @@
 WYCISZONY ADMINISTRACYJNIE
 GLOBALNIE WYCISZONY
-NIEUWIERZYTELNIONY
\ No newline at end of file
+NIEUWIERZYTELNIONY
+ZMIEŃ USTAWIENIA PRYWATNOŚCI
+Twój mikrofon jest wyłączony przez wybrane ustawienia prywatności.
+Ustawienia prywatności czatu głosowego
+Ta gra używa czatu głosowego. Użycie klawisza czatu głosowego (domyślnie "Q" i "V") zacznie przechwytywać twój mikrofon i transmitować głos do innych graczy na serwerze.\nNiektóre mechaniki gry mogą tymczasowo przechowywać krótkie nagrania twojego głosu, które zostaną automatycznie usunięte po rozłączeniu się z serwera.
+Odmów wszystko (wyłącz przechwytywanie mikrofonu)
```
Comment (zabszk) [inline Translations/pl/Voicechat.txt:8]:
```suggestion
Odmów wszystkiego (wyłącz przechwytywanie mikrofonu)
```

File: Translations/pl/Voicechat.txt [hunk 9]
```diff
@@ -1,3 +1,19 @@
...
+Personalizuj
+Akceptuj wszystko
+Akceptuj
+Odmów
+Przechwytywanie mikrofonu
+Pozwól na przechwytywanie twojego mikrofonu i jego transmitowanie do innych graczy podczas trzymania klawisza czatu głosowego. Podczas transmisji, ikonka mikrofonu pokaże się na dole ekranu. Twój głos nigdy nie będzie transmitowany bez twojej wiedzy. 
+Krótkie nagrania
+Pozwól na używanie krótkich nagrań twojego głosu. Obejmuje to umiejętność SCP-939, "Z Wieloma Głosami", która może nagrywać i odtwarzać krótkie nagrania głosów zabitych graczy. Wszystkie nagrania zostaną usunięte gdy opuścisz serwer lub runda dobiegnie końca.
```
Comment (zabszk) [inline Translations/pl/Voicechat.txt:16]:
```suggestion
Pozwól na używanie krótkich nagrań twojego głosu. Obejmuje to umiejętność SCP-939 "Z Wieloma Głosami", która może nagrywać i odtwarzać krótkie nagrania głosów zabitych graczy. Wszystkie nagrania zostaną usunięte, gdy opuścisz serwer lub runda dobiegnie końca.
```

File: Translations/pl/Voicechat.txt [hunk 10]
```diff
@@ -1,3 +1,19 @@
...
+Zastosuj
+Wróć
+Naciśnij "[key]" i wejdź w "Ustawienia" aby zmienić.
```
Comment (zabszk) [inline Translations/pl/Voicechat.txt:19]:
```suggestion
Naciśnij "[key]" i wejdź w "Ustawienia", aby zmienić.
```

File: Translations/pl/AttachmentEditors.txt [hunk 11]
```diff
@@ -14,3 +14,4 @@ WYŁĄCZANIE
 TRWA WYŁĄCZANIE STACJI ROBOCZEJ
 POKAŻ PODSUMOWANIE
 UKRYJ PODSUMOWANIE
+Personalizacja [Attachment]
```
Comment (zabszk) [inline Translations/pl/AttachmentEditors.txt:17]: Is the "[Attachment]" supposed to be untranslated?

**Review by zabszk — CHANGES_REQUESTED** [review]

File: Translations/pl/AttachmentEditors.txt (hunk 11 above)
Comment (MultiverseUncle) [reply Translations/pl/AttachmentEditors.txt:17]: The '[]' thingies suggest to me that [Attachment] is supposed to be a variable, similar to [key_attack], etc, so yeah, it should remain the same. Tho why it was made and where does it occur ingame is currently unknown (currently can't find it anywhere)

**Review by zabszk — DISMISSED** [review]

File: Translations/pl/SCP079_HUD.txt [hunk 12]
```diff
@@ -0,0 +1,67 @@
...
++[amount] EXP - Pomoc w zabiciu rebelii chaosu
++[amount] EXP - Pomoc w zabiciu człowieka
++[amount] EXP - Uświadczenie zabicia przez członka twojej drużyny
++[amount] EXP - Pomoc w pościgu członka twojej drużyny
++[amount] EXP - Obronienie członka twojej drużyny
++[amount] EXP - Zapobiegnięcie użycia Micro H.I.D.
+DOSTĘPNA KAMERA ONLINE
+ZGAŚ ŚWIATŁA W CAŁEJ STREFIE ([cost])
+$GASZENIE ŚWIATEŁ CAŁEJ STREFY$ jest dostępne - przycisk znajduje się obok jej nazwy na $mapie placówki$.
+TRWA ODNAWIANIE GASZENIE ŚWIATEŁ CAŁEJ STREFY
```
Comment (zabszk) [inline Translations/pl/SCP079_HUD.txt:60]:
```suggestion
TRWA ODNAWIANIE GASZENIA ŚWIATEŁ W CAŁEJ STREFIE
```

Typo and inconsistent with other strings.

File: Translations/pl/SCP079_HUD.txt [hunk 13]
```diff
@@ -0,0 +1,67 @@
...
++[amount] EXP - Pomoc w zabiciu Nine-Tailed Fox
++[amount] EXP - Pomoc w zabiciu rebelii chaosu
++[amount] EXP - Pomoc w zabiciu człowieka
++[amount] EXP - Uświadczenie zabicia przez członka twojej drużyny
++[amount] EXP - Pomoc w pościgu członka twojej drużyny
++[amount] EXP - Obronienie członka twojej drużyny
++[amount] EXP - Zapobiegnięcie użycia Micro H.I.D.
+DOSTĘPNA KAMERA ONLINE
+ZGAŚ ŚWIATŁA W CAŁEJ STREFIE ([cost])
+$GASZENIE ŚWIATEŁ CAŁEJ STREFY$ jest dostępne - przycisk znajduje się obok jej nazwy na $mapie placówki$.
```
Comment (zabszk) [inline Translations/pl/SCP079_HUD.txt:59]:
```suggestion
$GASZENIE ŚWIATEŁ W CAŁEJ STREFIE$ jest dostępne - przycisk znajduje się obok jej nazwy na $mapie placówki$.
```

Consistency.

File: Translations/pl/SCP079_HUD.txt [hunk 14]
```diff
@@ -0,0 +1,67 @@
...
++[amount] EXP - Pomoc w zabiciu naukowca 
++[amount] EXP - Pomoc w zabiciu ochroniarza
++[amount] EXP - Pomoc w zabiciu Nine-Tailed Fox
++[amount] EXP - Pomoc w zabiciu rebelii chaosu
++[amount] EXP - Pomoc w zabiciu człowieka
++[amount] EXP - Uświadczenie zabicia przez członka twojej drużyny
++[amount] EXP - Pomoc w pościgu członka twojej drużyny
++[amount] EXP - Obronienie członka twojej drużyny
++[amount] EXP - Zapobiegnięcie użycia Micro H.I.D.
+DOSTĘPNA KAMERA ONLINE
```
Comment (zabszk) [inline Translations/pl/SCP079_HUD.txt:57]: The translation doesn't really make any sense, offline camera can't be really available in this case.

Additionally I'm not sure whether the original string means that the CCTV camera is available or that the feed from the CCTV camera is now live on the screen.

**Review by zabszk — CHANGES_REQUESTED** [review]

File: Translations/pl/SCP079_HUD.txt (hunk 14 above)
Comment (MultiverseUncle) [reply Translations/pl/SCP079_HUD.txt:57]: Original string means that the feed from the camera is now live on the screen (aka 079's screen).
Btw, the same translated string was also used in old 079, but I guess I will just remove the "available" part from it.

**Review by zabszk — APPROVED** [review]

**Pull request merged**